src/features/pricing/winterStoragePricing/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { WinterStoragePricing as WinterStoragePricingData } from './__generated__/WinterStoragePricing';
+import { WinterStoragePrice } from './WinterStoragePricing';
+import { PeriodType } from '../../../@types/__generated__/globalTypes';
+
+export const getWinterStorageData = (data: WinterStoragePricingData | undefined | null): WinterStoragePrice[] => {
+  if (!data) return [];
+
+  return data.edges.reduce<WinterStoragePrice[]>((acc, edge) => {
+    if (!edge?.node) return acc;
+
+    const priceValue = edge.node.properties?.product?.priceValue;
+    const privateCustomer = Number.isNaN(priceValue) ? undefined : priceValue;
+    const company = Number.isNaN(priceValue) ? undefined : priceValue * 2;
```

Comment:
It would be nice to have this logic in BE

Reply:
I agree, but after [this discussion](https://helsinkicity.slack.com/archives/CBA1QHTHN/p1591270584115400), I learned that that `company` field is not needed on the backend and it's here only for better UX.